galaxy/Dockerfile

```diff
@@ -166,7 +167,8 @@ GALAXY_CONFIG_WELCOME_URL=$GALAXY_CONFIG_DIR/web/welcome.html \
 GALAXY_CONFIG_OVERRIDE_DEBUG=False \
 # We need to set $HOME for some Tool Shed tools (e.g Perl libs with $HOME/.cpan)
 HOME=$GALAXY_HOME \
-GALAXY_CONDA_PREFIX=$GALAXY_CONFIG_TOOL_DEPENDENCY_DIR/_conda
+GALAXY_CONDA_PREFIX=$GALAXY_CONFIG_TOOL_DEPENDENCY_DIR/_conda \
+PROXY_PREFIX=/
```

Comment:
Good point, I did it because I was checking the value there: https://github.com/galaxyproject/ansible-galaxy-extras/pull/127/files#diff-205c98235541487412061d12953dc4c0R52
But it would probably be wiser to remove this default value here and change the ansible startup.sh.j2 to:
`if [ "x$PROXY_PREFIX" != "x" ]`
I can make a PR there for this